sdk/monitor/monitor-opentelemetry-exporter/src/generated/models/index.ts

```diff
@@ -65,9 +65,13 @@ export interface MonitorBase {
  */
 export interface MonitorDomain {
   /**
-   * Ignored value.
+   * Describes unknown properties. The value of an unknown property can be of "any" type.
    */
-  test?: string;
+  [property: string]: any;
```

Comment:
This means anything that mixes in with `MonitorDomain` isn't type-checked by TS as all properties are valid. It would be best to avoid this if possible, since it's easy to make typos.

Reply:
This can go away once autorest supports `oneOf` polymophism. Until then, this is meant to be a temporary workaround